### ISSUE TITLE
[Bug] Census Geocode API Not Working

### ISSUE THREAD

**Issue by retacg** [move-coop/parsons#729]
When I attempted to use the Census Geocode API a few weeks ago I was unable to due to an error about the field names but the field names I used matched the documentation exactly 

## Detailed Description

I created a parsons table to pass to the`geocode_address_batch()` function and I received the following error. 
<img width="1440" alt="Screen Shot 2022-08-05 at 11 39 21 AM" src="https://user-images.githubusercontent.com/62267774/184727244-f8328843-b4ca-4be9-aa73-3cd66c11aaeb.png">

## To Reproduce

Here is the code I ran:
```
from parsons import Table, Redshift
from canalespy import setup_environment, logger
import os

CG=CensusGeocoder()
rs=Redshift()
tb = Table()

to_geocode=tb.from_redshift("""
select * from [schema].[table]""") 

CG.geocode_address_batch(to_geocode)
```

The table in redshift looks like this 
![Screen Shot 2022-08-15 at 4 18 53 PM](https://user-images.githubusercontent.com/62267774/184728208-304c6ed2-b0a7-4635-b1fe-421f864e9ab7.png)

## Your Environment

* Version of Parsons used (if you know): latest
* Environment name and version (e.g. Chrome 39, node.js 5.4): I dont know
* Operating System and version (desktop or mobile): Desktop

## Priority
Please indicate whether fixing this bug is high, medium, or low priority for you. If the issue is time-sensitive for you, please let us know when you need it addressed by.
Medium! It was VERY time consuming to geocode all those records via other means

**Comment by talevy42** [comment]
I believe this is purely a documentation issue, although it could be improved with some code to check field names and throw a more informative error. The correct set of field names should be:

- street
- city
- state
- zip

**Comment by mkrausse-ggtx** [comment]
I have a function I use for personal geocoding project that maps most column names to desired column names. We could also implement this. 

`def map_columns(df):
    # Create a list of common address column names
    address_columns = ['address', 'Address', 'ADDRESS', 'street', 'Street', 'STREET', 'addr', 'Addr', 'ADDR', 'location', 'Location', 'LOCATION', 'addr1']
    city_columns = ['city', 'City', 'CITY']
    state_columns = ['state', 'State', 'STATE']
    zip_columns = ['zip', 'Zip', 'ZIP', 'zip code', 'Zip Code', 'ZIP CODE', 'zipcode', 'Zipcode', 'ZIPCODE', 'ZIP5', 'zip5', 'Zip5']

    # Create a dictionary of column names and their corresponding common name
    address_dict = {k: 'address' for k in address_columns}
    city_dict = {k: 'city' for k in city_columns}
    state_dict = {k: 'state' for k in state_columns}
    zip_dict = {k: 'zip' for k in zip_columns}

    # Create a dictionary of all the dictionaries
    column_dict = {**address_dict, **city_dict, **state_dict, **zip_dict}

    # Rename the columns
    df.rename(columns=column_dict, inplace=True)

    return df`

**Comment by shaunagm** [comment]
Closed by #810
